assets/tests/components/propertiesPanel/minischedule.test.tsx

```diff
@@ -404,3 +405,51 @@ describe("MinischeduleBlock", () => {
     expect(tree).toMatchSnapshot()
   })
 })
+
+describe("BlockRow", () => {
+  test("Split breaks show as unpaid", () => {
+    const breakk: Break = {
+      breakType: "Split break",
+      startTime: 0,
+      endTime: 0,
+    }
+
+    const wrapper = mount(<BreakRow break={breakk} />)
+    expect(wrapper.html()).toContain("Break (Unpaid)")
+  })
+
+  test("Paid breaks show as paid", () => {
```

Comment:
@skyqrose would this satisfy codecov, if we did this instead here:

```
constPaidBreakTypes = ["Paid meal before", "Paid meal after", "Joinup", "Technical break"]

for(breakType in paidBreakTypes) {
  test(`Breaks of type ${breakType} are paid`, () => {
    const breakk: Break = {
      breakType: breakType,
      startTime: 0,
      endTime: 0
    }

    const wrapper = mount(<BreakRow break={breakk} />)
    expect(wrapper.html()).toContain("Break (Paid)")
  }
}
```

Reply:
When the source needs a hardcoded list or constant, I like to avoid duplicating it in the tests. But I think I found a way to abuse fall through order so that codecov doesn't care about getting all of the strings as long as each unique path is covered.